src/lib/organisms/methods.svelte

```diff
@@ -89,7 +89,7 @@
           {#if method.template && method.template.url}
             <picture>
               <img
-                src={method.template.url.replace(":webp", ":png")}
+                src={method.template.url.replace(":webp", ":jpg")}
```

Comment:
Shouldn't these images be gif's instead of jpg's?

Reply:
Uhmm, the images are not animations, so I think we should use PNG or JPG instead of GIF! And JPG would be a better option for optimization ..

Reply:
As you can see here below, I tested the network where you can see that JPG files load faster than PNG files.

> <img width="688" alt="Scherm­afbeelding 2024-05-06 om 10 37 20" src="https://github.com/fdnd-agency/visual-thinking/assets/112857932/6720a1cf-b663-4a8c-b829-8404f5a4d229">

> <img width="688" alt="Scherm­afbeelding 2024-05-06 om 10 39 28" src="https://github.com/fdnd-agency/visual-thinking/assets/112857932/a91f9f7c-fece-4ef2-a110-d5cc7e367c07">

Reply:
Ok check! I'll merge the request!